### DISCUSSION TITLE
where is the root path of SetVirtualHostNameToFolderMapping on VSTO?

### DISCUSSION THREAD

**Discussion by suyu0925** [MicrosoftEdge/WebView2Feedback#4004] [Q&A]
Webview2 is awesome, thanks for that, I can use react stack instead of unfamiliar WinForms controls.

But when I move frontend static webpage from server to local, I've hit a snag.
Here is my question: where is the root path or installation path of SetVirtualHostNameToFolderMapping on VSTO? I tried  %UserProfile%\Documents and %LocalAppData%\Apps\2.0\...\vsto_app(vsto installation path), none of them worked.

**Comment by victorthoang** [comment]
Hello @suyu0925,

I found something that might be similar in regard to when you move your static homepage from server to local.

[here](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2774). Let me know if this helps explain the possible issue!

Victor

**Comment by suyu0925** [comment]
Thanks for your reply, @victorthoang. 

But the issue you mentioned does not help, I created a [github repo](https://github.com/suyu0925/vsto-add-in-example) here to explain what I met.

[webView.CoreWebView2.SetVirtualHostNameToFolderMapping](https://github.com/suyu0925/vsto-add-in-example/blob/main/ExcelAddIn/LoginDlg.cs#L27) will throw System.IO.DirectoryNotFoundException when login button in ribbon is clicked.

**Comment by victorhuangwq** [comment]
May I ask if you are still facing this issue?
